src/event_loop.rs

```diff
@@ -422,19 +422,19 @@ impl<T> fmt::Display for EventLoopClosed<T> {
 
 impl<T: fmt::Debug> error::Error for EventLoopClosed<T> {}
 
-/// Filter controlling the propagation of device events.
+/// Controlling when device events are propagated.
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
-pub enum DeviceEventFilter {
-    /// Always filter out device events.
+pub enum DeviceEvents {
```

Comment:
the new name is a bit generic. does not convey what it is or what it is used for.

Reply:
I think @madsmtm original idea was to remove any mention of "filter" to avoid a double negative.
See https://github.com/rust-windowing/winit/issues/2591#issuecomment-1353220718.

Maybe it would be better to follow @ArturKovacs's advice and use `DeviceEventsAllowed` or `DeviceEventsCapture`?

Reply:
After further review I probably agree with this, would like to wait for a maintainers opinion on this though.